app/gui2/src/components/CircularMenu.vue

```diff
@@ -4,13 +4,13 @@ import ToggleIcon from '@/components/ToggleIcon.vue'
 
 const props = defineProps<{
   isOutputContextEnabledGlobally: boolean
-  isOutputContextOverridden: boolean
+  isRecordingOverridden: boolean
```

Comment:
shouldn't we be consistent in naming both these properties?

Reply:
ah, good catch, thanks